pkg/plugin/aggregation/run.go

```diff
@@ -154,3 +167,32 @@ func Cleanup(client kubernetes.Interface, plugins []plugin.Interface) {
 		p.Cleanup(client)
 	}
 }
+
+func sendUpdates(updater *status.Updater, ag *Aggregator) {
+	// Could have race conditions, but will be eventually consistent
+	for _, result := range ag.Results {
+		state := "complete"
+		if result.Error != "" {
+			state = "failed"
+		}
+		update := status.Plugin{
+			Node:   result.NodeName,
+			Plugin: result.ResultType,
+			Status: state,
+		}
+
+		if err := updater.Receive(&update); err != nil {
+			logrus.WithFields(
+				logrus.Fields{
+					"node":   update.Node,
+					"plugin": update.Plugin,
+					"status": state,
+					"at":     "sendUpdates",
+				},
+			).Error(err)
```

Comment:
I think this should be `.WithError(err).WithFields(...).Info("could not receive update")` since we don't really use the Error log level at all. At some point in the distant past we talked about logging and agreed [this logic was pretty good][log].

[log]: https://dave.cheney.net/2015/11/05/lets-talk-about-logging